nncf/experimental/common/tensor_statistics/collectors.py

```diff
@@ -31,15 +30,22 @@ class TensorReducerBase(ABC):
     the specified rule. Could handle tensors inplace or out of place.
     """
 
-    def __init__(self, reduction_shape: Optional[ReductionShape] = None, inplace: bool = False):
+    def __init__(
+        self, reduction_axes: Optional[ReductionAxes] = None, channel_axis: Optional[int] = None, inplace: bool = False
+    ):
```

Comment:
This change should be discussed as potentially ambiguous

Reply:
Please state exactly what is ambiguous about this.

Reply:
`channel_axis` is redundant, only `MeanPerChReducer` needs `channel_axis`

Reply:
The `MeanPerChReducer` defines a use case when the reduction axes are not completely known at the moment of instantiating the reducer, instead the requirement is "to reduce all axes except for a given axis (i.e. the channel axis)". Evidently, the code that instantiates reducers in general currently has no requirement that it should be always possible to get the shape of the input tensors to the reducer so that, in combination with a known channel axis, the equivalent reduction axes could be calculated and the reducer initialized with these. If you can suggest an exact way to refactor the existing code that instantiates `MeanPerChReducer` so that I could calculate the `reduction_axes` unambiguously there, I would be happy to remove the `channel_axis` init argument. I would also be able then to remove the `MeanPerChReducer` at all, since a regular `MeanReducer` with correct reduction axes would fulfill the role.